mappings/net/minecraft/entity/ai/control/BodyControl.mapping

```diff
@@ -1,2 +1,10 @@
 CLASS net/minecraft/class_1330 net/minecraft/entity/ai/control/BodyControl
+	FIELD field_6354 lastHeadYaw F
+	FIELD field_6355 ticksUpdated I
```

Comment:
activeTicks
```suggestion
	FIELD field_6355 activeTicks I
```

Reply:
Slap... I don't know if this is problems with updates or myself, but this name is no longer right in 1.17! Will submit a fix. :zany_face: